serverless/installing-openshift-serverless.adoc

```diff
@@ -12,8 +12,10 @@ include::modules/machineset-manually-scaling.adoc[leveloffset=+2]
 
 
 // Installing Service Mesh
-include::modules/serverless-installing-service-mesh-no-kiali-jaeger.adoc[leveloffset=+1]
+// Commented out for this release. May apply for future releases 
+// include::modules/serverless-installing-service-mesh-no-kiali-jaeger.adoc[leveloffset=+1]
 
+OpenShift Serverless requires an installed version of xref:../modules/ossm-operatorhub-install.adoc[Service Mesh].
```

Comment:
This `xref` doesn't build locally. I think the link needs to be to the correct assembly instead. Maybe:

> OpenShift Serverless requires an installed version of xref:../service_mesh/service_mesh_install/preparing-ossm-installation.adoc#preparing-ossm-installation[Service Mesh].